packages/web/components/PageHeader.tsx

```diff
@@ -41,63 +93,119 @@ export const PageHeader: React.FC = () => {
       justify="space-between"
       wrap="wrap"
       color="offwhite"
-      py="6"
+      py="4"
       px="8"
+      position="relative"
+      display={{ base: 'none', md: 'flex' }}
```

Comment:
If you remove this on the mobile version then there's no "Connect Wallet" button anymore. Not a fan of having two separate components for this reason, should try to make a single AppDrawer component that automatically updates its layout for mobile.